test/WebpackConfig.js

```diff
@@ -940,24 +940,15 @@ describe('WebpackConfig object', () => {
         it('Adds new externals', () => {
             const config = createConfig();
 
-            expect(config.externals).to.deep.equals({});
+            expect(config.externals).to.deep.equals([]);
 
             config.addExternals({ 'jquery': 'jQuery', 'react': 'react' });
             config.addExternals({ 'lodash': 'lodash' });
 
-            expect(config.externals).to.deep.equals({
-                'jquery': 'jQuery',
-                'react': 'react',
-                'lodash': 'lodash'
-            });
-        });
-
-        it('Calling it with an invalid argument', () => {
```

Comment:
Shouldn't we keep that test and check for the types that are actually accepted by `config.externals` in Webpack?

Reply:
My thought here was to just let Webpack report an error with any invalid externals. From what I can tell though, if the external is not a string, object, regex, or function webpack just ignores it.

Reply:
Yeah maybe that's not needed... since it accepts nearly everything it's unlikely someone will try to pass it something invalid.